src/views/Home.vue

```diff
@@ -1,12 +1,37 @@
 <template>
   <div>
     <Header :id="restaurant.id" :title="restaurant.title" :slug="restaurant.slug" :logo="restaurant.logo" :banner="restaurant.banner"></Header>
+    <div class="todays-special-wrapper">
+      <div class="categories" v-for="category in categories" :key="category">
+        {{ category }}
+      </div>
+    </div>
     <HorizontalMenu title="Today's Special" :isLoading="isLoadingSpecials" :dishes="todaysSpecial"></HorizontalMenu>
-    <AccordionMenu title="Menu" :dishes="dishes" :courses="courses" :isLoading="isLoadingMenu"></AccordionMenu>
+    <SearchBar
+      :dishes="dishes"
+      @filter-dish="filterByText"
+      @clear-filter="clearFilter"
+    >
+    </SearchBar>
+    <h2>Menu</h2>
+    <div class="filtered-items" ref="filteredItems" v-show="filteredDishes.length > 0">
+      <AccordionMenu :dishes="filteredDishes" :courses="courses" :isLoading="isLoadingMenu" :activeByDefault="true"></AccordionMenu>
+    </div>
+    <div class="all-items" v-show="filteredDishes.length == 0">
+      <AccordionMenu :dishes="dishes" :courses="courses" :isLoading="isLoadingMenu" :activeByDefault="false"></AccordionMenu>
```

Comment:
You're missing the title from the section (:title="Menu")

Reply:
We shouldn't put titles of sections into components, it prevents them from being reusable.

Reply:
That is a very good point. I will mark as solved